Sources/StreamChat/Controllers/MemberListController/MemberListController.swift

```diff
@@ -95,9 +95,8 @@ public class ChatChannelMemberListController: DataController, DelegateCallable,
         )
     }
 
-    private func createMemberListObserver() -> ListDatabaseObserverWrapper<ChatChannelMember, MemberDTO> {
+    private func createMemberListObserver() -> BackgroundListDatabaseObserver<ChatChannelMember, MemberDTO> {
         let observer = environment.memberListObserverBuilder(
-            false,
```

Comment:
You can see here, that the value of `isBackgroundMapping` is hardcoded to `false`

Reply:
Changed it back